theme-data/dark/dark-common.json

```diff
@@ -308,6 +308,30 @@
           "disabled": "@color-black-alpha-40"
         }
       }
+    },
+    "chips": {
+      "multiline": {
```

Comment:
we do not need to add theme tokens for chips multiline (especially because they do not exist in the Figma theme tokens). In the component library, they are using Text/TextTeam tokens, which are already added in our tokens repo here https://github.com/momentum-design/tokens/blob/32e42ecc7f5fd7122eaec50f408f41a1a4e94457/reference-tokens/color-theme-additional.json#L5
<img width="750" alt="image" src="https://user-images.githubusercontent.com/56999622/172363995-3aad1e08-c325-47e5-aba1-f781c7e3d758.png">